fastify.js

```diff
@@ -826,7 +826,13 @@ function build (options) {
       message: 'Client Error',
       statusCode: 400
     })
-    log.error({ err }, 'client error')
```

Comment:
I think we can log all errors here as debug instead of checking for specific type of error

Reply:
i have no problem with that, should i change it?

Reply:
Let's wait for @mcollina's opinion

Reply:
Just found out the there can be another error type about this problem, so maybe cemremengu suggestion is a better solution, instead of filtering many error types.

```
ERROR [2019-01-04 10:10:31.594 +0000] (15728 on PredatorRTW): client error
    err: {
      "type": "NodeError",
      "message": "TLS handshake timeout",
      "stack":
          Error [ERR_TLS_HANDSHAKE_TIMEOUT]: TLS handshake timeout
              at TLSSocket._handleTimeout (_tls_wrap.js:594:22)
              at Object.onceWrapper (events.js:273:13)
              at TLSSocket.emit (events.js:182:13)
              at TLSSocket.Socket._onTimeout (net.js:452:8)
              at listOnTimeout (timers.js:324:15)
              at processTimers (timers.js:268:5)
    }
```